File: SolvedKatas/Java/solvedkatas/src/main/java/com/solvedkatas/java/solvedkatas/MultipleFiveThree.java
package com.solvedkatas.java.solvedkatas;

import java.util.stream.IntStream;

public class MultipleFiveThree {

    public int solution(int number) {
        return IntStream.range(3, number).filter(result -> result % 3 == 0 || result % 5 == 0).sum();
    }
}

File: SolvedKatas/CSharp/SolvedKatas/SolvedKatas/SolvedKatas/MultipleFiveThree.cs
using System.Linq;

namespace SolvedKatas
{
    public static class MultipleFiveThree
    {
        public static int Solution(int value)
        {
            return Enumerable.Range(0, value).Where(x => x % 3 == 0 || x % 5 == 0).Sum();
        }
    }
}